src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -178,6 +178,12 @@ private function registerCommonConfiguration(ContainerBuilder $container, array
         $container->setParameter('api_platform.title', $config['title']);
         $container->setParameter('api_platform.description', $config['description']);
         $container->setParameter('api_platform.version', $config['version']);
+        $container->setParameter('api_platform.termsOfService', $config['termsOfService']);
```

Comment:
Maybe that we should put this under an `openapi` node?

Reply:
I also had this idea but reverted it because i think if we add an openapi section we should also move title, description and version there to have it consistent.  wdyt ?

Reply:
Some are used by the Hydra documentation normalizer also. As these new ones are openapi specifics I'd add them under an open api node.

Reply:
You are right. I moved only the new ones into an openapi section.